loops.js

```diff
@@ -24,7 +26,15 @@
 // loop...
 // klaar!
 // ==========================================
+for(let i=1 ; i <= 5 ;i++){
+    if(i === 5){
+        console.log("klaar");
+    } else {
```

Comment:
De implementatie om scores om te zetten in lettercijfers is correct. De if-else structuur is logisch en goed opgebouwd.